ui-particles-angular/projects/ui-particles-angular/src/lib/gio-save-bar/gio-save-bar.component.html

```diff
@@ -18,9 +18,9 @@
 
 <div *ngIf="isOpen()" [@slideUpDown] [@.disabled]="creationMode">
   <mat-card class="save-bar__content" [class.mat-elevation-z3]="creationMode" [class.mat-elevation-z6]="!creationMode">
```

Comment:
Can we have the elevation level "mat-elevation-z5" in both cases?

```suggestion
  <mat-card class="save-bar__content mat-elevation-z5">
```

Reply:
As discussed we have 2 different elevations to match one of the cards when we are in a creation form. Before updating this we need to ensure every cases are well handled. Same thing if we want the default elevation to be 5, it needs to be aligned with the cards.